### PR TITLE
Less frequent sanitizer runs

### PR DESCRIPTION
**Pull request by boeschf**
Reduce CI times by only running the sanitizer tests when we commit to master and once a day.

### CODE REVIEW

**Comment by thorstenhater** [comment]
That might be more often than now, actually ;)

**Comment by boeschf** [comment]
true :) Once a week?

**Comment by thorstenhater** [comment]
Yeah. Can you make it so that we get an automatically created issue?

**Comment by brenthuisman** [comment]
For Issue creation from e.g. a log file: https://github.com/marketplace/actions/create-issue-from-file

File: .github/workflows/sanitize.yml [hunk 1]
```diff
@@ -58,7 +62,35 @@ jobs:
...
-          build/bin/unit --gtest_filter=-*DeathTest
-          build/bin/unit-modcc
+          build/bin/unit --gtest_filter=-*DeathTest 2>&1 | tee output.log
+          build/bin/unit-modcc 2>&1 | tee -a output.log
+        shell: bash
       - name: Run examples
-        run: scripts/run_cpp_examples.sh
+        run: scripts/run_cpp_examples.sh 2>&1 | tee -a output.log
+        shell: bash
+      - name: Try to fail
```
[the remark this answers is not in the document]
Comment (boeschf) [reply .github/workflows/sanitize.yml:71]: debugging - will remove once it generates the output I want

[the remark this answers is not in the document]
Comment (brenthuisman) [reply .github/workflows/sanitize.yml:74]: You can mark the PR draft if it's not ready for review.

[the remark this answers is not in the document]
Comment (boeschf) [reply .github/workflows/sanitize.yml:74]: I know, just wanted to exclude any potential side-effects.

**Pull request closed without merging**